client/components/AttributionEditor/AttributionDetailControls.tsx

```diff
@@ -129,8 +129,6 @@ const AttributionDetailControls = (props: Props) => {
 				/>
 				{isShadowAttribution && (
 					<InputField
-						// @ts-expect-error ts-migrate(2322) FIXME: Type '"" | Element | undefined' is not assignable ... Remove this comment to see the full error message
-						rightElement={orcid && <Tag minimal>ORCID</Tag>}
```

Comment:
The other type changes make sense to me, but is this one intentional too? And do you understand what was happening here before you removed it because I do not (why pass this unexpected `rightElement` prop)?

Also just a note that this throws an error now bc of the unused `Tag` import

Reply:
You're right, I incorrectly thought this was like a vestigial prop as maybe this was changed from being an `InputGroup` to an `InputField`. 

turns out i am the one who changed that lmao haha.
I wanted to use the `error` field on the `InputField`, as `InputGroup` does not have that, in doing so unknowingly removing that `Tag`, which ofc did not throw an error bc it was being `ts-expect-error`ed, although for completely different reasons after changing the type of component.

You can see rn on prod that it does not display a `Tag`
<img width="977" alt="image" src="https://github.com/pubpub/pubpub/assets/21983833/bf85352a-6d94-4fa1-b0de-8bbdb1383262">

whereas it "should" look like this (this branch now)

<img width="1061" alt="image" src="https://github.com/pubpub/pubpub/assets/21983833/760b0f1d-53dd-48db-a711-64f084cf2c79">

I reverted it to being an `InputGroup` instead and fixed the type error another way, and manually implemented some error showing logic for this component.

Errors look like this on prod
<img width="982" alt="image" src="https://github.com/pubpub/pubpub/assets/21983833/8a65149c-7a52-4f25-a400-6d9d1ddc2951">

And now look like this on this branch
<img width="1068" alt="image" src="https://github.com/pubpub/pubpub/assets/21983833/a15ed130-9be8-487a-b1f8-920950b3b7f7">

I guess the font size is slightly different, other than that it behaves the same!

Again, thanks for mentioning this. I should have pointed it out in the first place, but this way I actually double checked why I made that change haha

Reply:
thank you for the detailed explanation!! gabeN bless